### PR TITLE
[CI/CD]Add topology and device type argument for PR test

### PR DESCRIPTION
**Pull request by lerry-lee**
### Description of PR

[CI/CD]Add topology and device type argument for PR test. 
If PR test, add argument `--topology=t0,any` for `t0`, `t0-64-32`, and `dualtor` topologies. And add `--topolog=t1,any` for `t1-lag` and `t1-8-lag` topologies. Add `--device_type=vs` argument for all PR test scritps.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

### CODE REVIEW

File: .azure-pipelines/test_plan.py [hunk 1]
```diff
@@ -223,6 +223,20 @@ def create(self, topology, test_plan_name="my_test_plan", deploy_mg_extra_params
...
         common_extra_params = common_extra_params + " --completeness_level=confident --allow_recover"
 
+        t0_topos = ["t0", "t0-64-32", "dualtor"]
+        t1_topos = ["t1-lag", "t1-8-lag"]
+
+        # Add topo and device type args for PR test
+        if test_plan_type == "PR":
+            # Add topo arg
+            if topology in t0_topos:
+                common_extra_params = common_extra_params + " --topology=t0,any"
```
[the remark this answers is not in the document]
Comment (lerry-lee) [reply .azure-pipelines/test_plan.py:233]: yes

[the remark this answers is not in the document]
Comment (lerry-lee) [reply .azure-pipelines/test_plan.py:233]: dualtor is also t0 topo

[the remark this answers is not in the document]
Comment (lerry-lee) [reply .azure-pipelines/test_plan.py:233]: sure, will add dpu topo

[the remark this answers is not in the document]
Comment (yutongzhang-microsoft) [reply .azure-pipelines/test_plan.py:233]: No, there is also a mark `pytest.mark.topology("dualtor")`

[the remark this answers is not in the document]
Comment (yutongzhang-microsoft) [reply .azure-pipelines/test_plan.py:233]: For dualtor, you need three marks

t0
dualtor 
any

**Comment by yutongzhang-microsoft** [comment]
What about multi-asic, t0-sonic. t0-2vlan?

**Comment by lerry-lee** [comment]
> What about multi-asic, t0-sonic. t0-2vlan?

the topo of **multi-asic** is `t1-8-lag`, **t0-sonic** is `t0-64-32`, **t0-2vlan** is `t0`

**Review by yutongzhang-microsoft — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**